solver_generation/generate_solver.m

```diff
@@ -66,10 +66,13 @@
 codeoptions.solvemethod = 'PDIP_NLP'; 
 codeoptions.solver_timeout = 1;
 
+%Add to generate code for ForcesPro version 5
+codeoptions.legacy_interface = 1;
+
 codeoptions.avx = 1;
 codeoptions.sse = 1;
 
-codeoptions.platform = 'Gnu-x86_64';
+%codeoptions.platform = 'Gnu-x86_64'; %comment out if only use for simulation

```

Comment:
In general people will test this solver in their laptops, and only change the platform when they know what they do. If there is no conflict with the new version I prefer to leave this as before.